spectra-generator/src/test/java/se/de/hu_berlin/informatik/gen/spectra/spectra/TestProjects.java

```diff
@@ -97,13 +97,13 @@ public Lang8b() {
 
     public static class Lang10b extends AbstractTestProject {
 
-        private static final String testCP = getStdResourcesDir() + File.separator + "Lang10b/target/classes" + File.pathSeparator
-                + getStdResourcesDir() + File.separator + "Lang10b/target/tests" + File.pathSeparator
-                + getStdResourcesDir() + File.separator + "Lang10b/lib/junit-4.11.jar" + File.pathSeparator
-                + getStdResourcesDir() + File.separator + "Lang10b/lib/easymock.jar" + File.pathSeparator
-                + getStdResourcesDir() + File.separator + "Lang10b/lib/commons-io.jar" + File.pathSeparator
-                + getStdResourcesDir() + File.separator + "Lang10b/lib/cglib.jar" + File.pathSeparator
-                + getStdResourcesDir() + File.separator + "Lang10b/lib/asm.jar";
+        private static final String testCP = getStdResourcesDir() + File.separator + "Lang10b\\target\\classes" + File.pathSeparator
+                + getStdResourcesDir() + File.separator + "Lang10b\\target\\tests" + File.pathSeparator
+                + getStdResourcesDir() + File.separator + "Lang10b\\lib\\junit-4.11.jar" + File.pathSeparator
+                + getStdResourcesDir() + File.separator + "Lang10b\\lib\\easymock.jar" + File.pathSeparator
+                + getStdResourcesDir() + File.separator + "Lang10b\\lib\\commons-io.jar" + File.pathSeparator
+                + getStdResourcesDir() + File.separator + "Lang10b\\lib\\cglib.jar" + File.pathSeparator
+                + getStdResourcesDir() + File.separator + "Lang10b\\lib\\asm.jar";
```

Comment:
these should probably be substituted by propoer system-specific path dividers... So replace with File.pathSeparator, I would think.